.travis.yml

```diff
@@ -1,26 +1,39 @@
 language: python
 
+python:
```

Comment:
Adding this will create additional entries in the build matrix. See also [the ci build](https://travis-ci.org/Bouke/django-airbrake/builds/350670567?utm_source=github_status&utm_medium=notification). No need to add this list of python versions, and control the build matrix using the `matrix:` field.